queries/systemverilog/highlights.scm

```diff
@@ -0,0 +1,565 @@
+  (simple_identifier)? @label)
+
+(covergroup_declaration
+  .
+  name: (simple_identifier) @constructor
+  (simple_identifier)? @label)
+
+(package_declaration
+  .
+  name: (simple_identifier) @constructor
+  (simple_identifier)? @label)
+
+(checker_declaration
+  .
+  name: (simple_identifier) @constructor
+  (simple_identifier)? @label)
+
+(interface_declaration
+  .
+  [
+    (simple_identifier) @constructor
+    (interface_nonansi_header
+      (simple_identifier) @constructor)
+    (interface_ansi_header
+      (simple_identifier) @constructor)
+  ]
+  (simple_identifier)? @label)
+
+(module_declaration
+  .
+  [
+    (simple_identifier) @constructor
+    (module_nonansi_header
+      (simple_identifier) @constructor)
+    (module_ansi_header
+      (simple_identifier) @constructor)
+  ]
+  (simple_identifier)? @label)
+
+(program_declaration
+  .
+  [
+    (simple_identifier) @constructor
+    (program_nonansi_header
+      (simple_identifier) @constructor)
+    (program_ansi_header
+      (simple_identifier) @constructor)
+  ]
+  (simple_identifier)? @label)
+
+; function.call
+(method_call_body
+  name: (simple_identifier) @function.call)
+
+(tf_call
+  (hierarchical_identifier
+    (simple_identifier) @function.call))
+
+; instance
+(module_instantiation
+  instance_type: (simple_identifier) @constructor)
+
+(name_of_instance
+  instance_name: (simple_identifier) @module)
+
+(sequence_instance
+  (hierarchical_identifier
+    (simple_identifier) @module))
+
+(udp_instantiation
+  (simple_identifier) @constructor)
+
+(ansi_port_declaration
+  (interface_port_header
+    interface_name: (simple_identifier) @variable
+    modport_name: (simple_identifier) @variable.member)
+  port_name: (simple_identifier) @variable)
+
+; bind
+(bind_directive
+  (bind_target_scope
+    (simple_identifier) @constructor))
+
+(bind_target_instance
+  (hierarchical_identifier
+    (simple_identifier) @module))
+
+; assertion
+(concurrent_assertion_item
+  (simple_identifier) @label)
+
+; converge
+(cover_point
+  name: (simple_identifier) @label)
+
+(cover_cross
+  name: (simple_identifier) @module)
+
+(list_of_cross_items
+  (simple_identifier) @constructor)
+
+;package
+(package_import_item
+  (simple_identifier) @constructor)
+
+; label
+(seq_block
+  (simple_identifier) @label)
+
+(statement
+  block_name: (simple_identifier) @label)
+
+; dpi
+(dpi_spec_string) @string
+
+c_name: (c_identifier) @function
+
+(dpi_import_export
+  name: (simple_identifier) @function)
+
+; type def
+(class_type
+  (simple_identifier) @constructor)
+
+(class_type
+  (simple_identifier)
+  (simple_identifier) @type)
+
+(data_type
+  (class_scope
+    (class_type
+      (simple_identifier) @constructor)))
+
+(task_prototype
+  name: (simple_identifier) @function)
+
+(function_prototype
+  name: (simple_identifier) @function)
+
+(type_assignment
+  name: (simple_identifier) @type.definition)
+
+(enum_name_declaration
+  (simple_identifier) @type.parameter)
```

Comment:
This is not a legal capture; you can only use the ones listed in CONTRIBUTING.md.